### ISSUE TITLE
Typescript keeps autocompleting the first suggestion when I type `this` as soon as I press the dot character.

### ISSUE THREAD

**Issue by pastcompute** [Microsoft/vscode#72893]
Issue Type: <b>Bug</b>

1. Edit a typescript file
2. Anywhere, type `this.`
3. Visual studio code will autocomplete some pseudorandom text, although I suspect it might just be the first in alphabetical order.

I've noted over the last month or this strange behaviour that didn't use to happen, and it is very annoying, I have to keep deleting whatever it puts there and only then it pauses and lets me decide what to autocomplete!

I've read through some other issues around autocomplete and they don't seem to be doing the same as this bizarre behaviour... it could be related to this https://github.com/Microsoft/vscode/issues/60552 but surely it should wait until I've chosen from the list of available suggestions before using the dot!

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:22:55.268Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz (4 x 2600)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.77GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
overtype|ada|0.2.0
Bookmarks|ale|10.4.3
insert-unicode|bru|0.6.0
vscode-modelines|chr|2.0.2
typescript-hero|rbb|3.0.0
move-ts|str|1.11.3
sort-lines|Tyr|1.8.0
markdown-all-in-one|yzh|2.2.0

</details>

**Comment by pastcompute** [comment]
umm.. which issue is the duplicate then? This is not a useful bot if it doesn't provide such basic information. I did use the search first... there are 5700 open issues, and if I add autocomplete this "narrows" it down to 94. I found https://github.com/Microsoft/vscode/issues/71795, but that has also been closed by this bot without providing further information. (Sometimes AI is not always that "I", this might be one of those cases ;)
